bin/evaluate

```diff
@@ -13,7 +13,7 @@ import argparse
 import warnings
 
 from madmom.utils import search_files, match_file
-from madmom.evaluation import onsets, beats, notes, tempo, alignment
+from madmom.evaluation import onsets, beats, notes, tempo, alignment, chords
```

Comment:
I started re-arranging the imports alphabetically, because they are easier to read and searchable. Although not explicitly stated in PEP8, this is also recommended by various Python style guides. You could also do so when adding a final commit...